src/main.cpp

```diff
@@ -33,6 +49,10 @@ int main()
 
         window.clear();
         window.draw(sprite);
+        if (font.loadFromFile("resources/OpenSans-Regular.ttf"))
```

Comment:
Loading stuff from the disk is a relatively heavy operation and here you are trying to load the same font on every frame. This will result in a lot of expensive processing, and is unnecessary too, because all you want to know is if the font was loaded successfully earlier. That can be achieved with even a simple boolean variable.